canvas_workflow_helpers/protocols/example_protocols/breast_cancer_screening.py

```diff
@@ -0,0 +1,311 @@
+        mammography in women 75 years and older (I statement).
+
+        U.S. Preventive Services Task Force (2009)
+        Grade: B recommendation. The USPSTF recommends biennial screening mammography for women
+        aged 50 to 74 years.
+        Grade: C recommendation. The decision to start regular, biennial screening mammography
+        before the age of 50 years should be an individual one and take patient context into
+        account, including the patient's values regarding specific benefits and harms.
+        Grade: I Statement. The USPSTF concludes that the current evidence is insufficient to
+        assess the additional benefits and harms of screening mammography in women 75 years or
+        older.
+        Grade: D recommendation. The USPSTF recommends against teaching breast self-examination
+        (BSE).
+        Grade: I Statement. The USPSTF concludes that the current evidence is insufficient to
+        assess the additional benefits and harms of clinical breast examination (CBE) beyond
+        screening mammography in women 40 years or older.
+        Grade: I Statement. The USPSTF concludes that the current evidence is insufficient to
+        assess the additional benefits and harms of either digital mammography or magnetic
+        resonance imaging (MRI) instead of film mammography as screening modalities for breast
+        cancer.
+        """
+        result = ProtocolResult()
+
+        if self.in_denominator():
+            first_name = self.patient.first_name
+            extra_months = 0 if self.period_adjustment else self.EXTRA_SCREENING_MONTHS
+            if self.in_numerator() and self._on_date:
+                result.due_in = (
+                    self._on_date.shift(
+                        days=self.timeframe.duration, months=extra_months
+                    )
+                    - self.now
+                ).days
+                result.status = STATUS_SATISFIED
+                result.add_narrative(
+                    f"{first_name} had a mammography {self.display_date(self._on_date)}."
+                )
+            else:
+                result.due_in = -1
+                result.status = STATUS_DUE
+                result.add_narrative("No relevant exams found.")
+                result.add_narrative(self.screening_interval_context())
+                result.add_instruction_recommendation(
+                    key="PLAN_RECOMMENDATION",
+                    rank=1,
+                    button="Plan",
+                    patient=self.patient,
+                    instruction=Mammography,
+                    title="Discuss breast cancer screening",
+                )
+                result.add_imaging_recommendation(
+                    key="IMAGING_RECOMMENDATION",
+                    rank=2,
+                    button="Order",
+                    patient=self.patient,
+                    imaging=MammographyImaging,
+                    title="Order screening mammogram",
```

Comment:
I forgot about the bug in this recommendation in that it searches in ImagingReportTemplate model instead of making a call to our shared services. So to make this look closer to the order can you change this line to 
```suggestion
                    title="Order Mammography, screening; bilateral",
```